man/fi_cntr.3.md

```diff
@@ -217,9 +217,9 @@ return from this call, the counter will be greater than or equal to
 the input threshold value.
 
 If an operation associated with the counter encounters an error, it
-will increment the error value associated with the counter.  Any
-change in a counter's error value will unblock any thread inside
-fi_cntr_wait.
+will increment the error value associated with the counter.  A counter's
+wait object will be signaled if the counter's current error value
+differs from the last value reported by fi_cntr_readerr.
```

Comment:
> Need to think on this a bit ... can "last value" be confusing in a multi-
> threaded scenario?

From the perspective of the provider, I don't think so.  The app will need to deal with synchronization if it uses multiple threads to read the error counter and then wants to wait from one or more threads.